kernel/packages/shared/profiles/sagas.ts

```diff
@@ -387,15 +403,26 @@ function* populateFaceIfNecessary(profile: any, resolution: string) {
 }
 
 export async function profileServerRequest(serverUrl: string, userId: string) {
-  try {
-    const request = await fetch(`${serverUrl}/${userId}`)
-    if (!request.ok) {
-      throw new Error('Profile not found')
-    }
-    return await request.json()
-  } catch (up) {
-    throw up
+  const request = await fetch(`${serverUrl}/${userId}`)
+  if (!request.ok) {
+    throw new Error('Profile not found')
   }
+  return request.json()
+}
+
+export function* createSignUpProfile(profile: Profile, identity: ExplorerIdentity) {
```

Comment:
We already talked about this, but ensure to use saveProfileRequest so we minimize alternative flows!